solidity/migrations/2_deploy_contracts.js

```diff
@@ -62,30 +62,24 @@ module.exports = async function(deployer, network) {
   );
 
   let oldStakingContractAddress;
-  if (network === 'mainnet') {
-    // v1.0.1 TokenStaking contract address
-    oldStakingContractAddress = '0x6D1140a8c8e6Fac242652F0a5A8171b898c67600';
-  } else if (network === 'ropsten') {
-    // 1.3.0-rc.0 TokenStaking contract address
-    oldStakingContractAddress = '0x8117632eC1D514550b3880Bc68F9AC1A76c9C67B';
-  } else {
+  if (network === 'local' || network === 'ropsten') {
```

Comment:
For ropsten should we consider using already deployed contracts similiar as we will do for mainnet?

Reply:
It's been explained in https://github.com/keep-network/keep-core/pull/1972/commits/b2aa91b9a4637d0c4f206d2028cad8aaf67cab4d:

```
We can't just point to the already deployed TokenStaking version on
Ropsten because it would require using the same KeepToken and TokenGrant
contracts and doing that is non-trivial given how our workflows are
configured: if we don't deploy a contract, it's not included in NPM
package and if it's not included in NPM package, provisioning scripts
and dashboard may not work.

Given that we are going to test copy stake flow using mainnet version of
contracts manually, we are deploying OldTokenStaking stub to allow for
UI/UX tests for copy flow on Ropsten but we don't consider those UI/UX
tests as ones giving us any guarantee the whole functionality works as
expected. To have this guarantee, we are going to test copy stake flow
using a clone of mainnet contracts.
```